### ISSUE TITLE
Draggable screenshot indicator

### ISSUE THREAD

**Issue by ottopiramuthu** [Jelmerro/Vieb#311]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Addition or change**
The screenshot indicator is now only changed when the pixel values in an entered command are changed. It would be nice if you could drag the visible rectangle with the mouse to make quick adjustments to the screenshot region.

**Alternatives considered**
Changing the w,h,x,y values can be a lot of trial and error.

**Comment by ottopiramuthu** [comment]
I realize now that my current suggestion would make it necessary to update the command in the navbar when the mouse updates the rectangle. Maybe this could be fixed by instead passing an "interactive" argument to screenshot. So something like
```
:screenshot interactive s.png
```
would let the user resize a screenshot indicator with the mouse. Once they are done resizing, they could press `<Enter>` and `screenshot` would be called with the given rectangle coordinates.

Maybe if rectangle coordinates are given with the interactive argument, they could determine the starting position of the rectangle indicator. For example,
```
:screenshot interactive 50,50,50,50 s.png
```
would start with a mouse-draggable square of size 50 at 50,50.

Let me know if you have any other ideas.

**Comment by Jelmerro** [comment]
Let me know if this works as expected in the latest commit.

**Comment by ottopiramuthu** [comment]
This works mostly as expected. Two things:
- When I try dragging from a point on the border of the indicator, sometimes the indicator jumps (but then it can be dragged as normal, just with an offset)
- It would be nice if there was also a way to resize the indicator with the mouse. Maybe holding the right mouse button could resize it? I'm open to suggestions.

**Comment by Jelmerro** [comment]
I can't reproduce the first thing, but it's probably caused by the translation I had to do for mouse events of the page to the frame. I'll keep my eyes open for cases where this might happen, one of which that comes to mind now is pages which are not at the default zoom level.

I made it so that other mouse button dragging changes the width and height instead of the position. This takes care of your second point.
